web/web/notifications/static/actions.js

```diff
@@ -39,6 +39,37 @@ export function getNotifications() {
     }
 }
 
+export function postNotifications(data) {
+    return dispatch => {
+        api.post('notification', { json: { ...data } }, (response) => {
```

Comment:
Not sure where to place this comment but this could also be a future issue, or if you want to address it now. The current logged in user should only see notifications on this page that are created by them. So just filtering the get_notifications so it's only displaying the ones created by the logged in user. (This connects back to the comment about connected the created_by id to the current logged in user). Thanks Derin! Looks great overall.

Reply:
fixed

Reply:
Added a filter in getNotifications() action creator to fix this